README.rst

```diff
@@ -69,3 +69,11 @@ Compatibility
 
 This library should be compatible with py-bcrypt and it will run on Python
 2.6+, 3.2+, and PyPy.
+
+Security
+--------
+
+``bcrypt`` follows the `same security policy as cryptography`_, if you
+identify a vulnerability, we ask you to contact us privately.
+
+.. _`same security policy as cryptography`: https://cryptography.io/en/latest/security/
```

Comment:
Have you thought about using something like https://hackerone.com/ (or some of the other platforms/services out there) to help with this? Just curious... (disclaimer: speaking as a security engineer here who likes to see projects handling security reports better, not as hackerone.com employee).